ui/app/styles/helper-classes/flexbox-and-grid.scss

```diff
@@ -46,14 +46,18 @@
   flex-direction: row;
 }
 
-.has-gap-m {
+.gap-16 {
```

Comment:
love these new classnames; it's nice to know exactly what # of pixels they're referring to